share/functions/vi_increment_decrement.fish

```diff
@@ -0,0 +1,34 @@
+alias vi_dec 'vi_increment_decrement subtract'
+alias vi_inc 'vi_increment_decrement add'
+
+function vi_increment_decrement --description 'increment or decrement numbers with ctrl+x ctrl+a'
+    set lnum (commandline --cursor)
+    set lstr (commandline -b)
```

Comment:
Would you mind using long options in scripts, please? Short options should only be used in command line usage, not when other people read the code.